test/quota.QuotaManager.spec.js

```diff
@@ -173,6 +173,7 @@ describe('quota', () => {
       const smallPlanId = 'bc158c9a-7934-401e-94ab-057082a5073e';
       const xsmallPlanId = 'bc158c9a-7934-401e-94ab-057082a5073f';
       const smallPlanGuid = 'c77ff479-ea63-42ea-b6d1-d824c4012f1c';
+      const rabbitmqVirtualHostPlanGuid = 'd035f948-5d3a-43d7-9aec-954e134c3e9d';
```

Comment:
better to rename the variable and have a name independent of rabbitmq.

`skipQuotaCheckPlanGuid` ?

also just to understand , this plan is not defined in the test ? like it does not have metadata attribute defined ?